.github/test-infra/aws/rke2/scripts/get-kubeconfig.sh

```diff
@@ -32,7 +32,9 @@ scp -o StrictHostKeyChecking=no -i key.pem ${node_user}@${bootstrap_ip}:/home/${
 
 # Replace the loopback address with the cluster hostname
 sed -i "s/127.0.0.1/${bootstrap_ip}/g" ./rke2-config > /dev/null
-export KUBECONFIG=$(pwd)/rke2-config
+mkdir -p /home/runner/.kube
+mv ./rke2-config /home/runner/.kube/config
+#export KUBECONFIG=$(pwd)/rke2-config
```

Comment:
I think this is fine, just noting that it would no longer work as expected locally I think, unless someone exported KUBECONFIG on their own.

Reply:
Good callout - I am doing the same thing here: https://github.com/defenseunicorns/uds-core/blob/78e09d917a4b340eaa136d693f1c301286ebc626/.github/test-infra/azure/aks/uds-config.tf#L32

Considering that there's multiple perquisites to deploying any of this TF locally, we can capture this in a separate document. Opened this [issue](https://github.com/defenseunicorns/uds-core/issues/923) to capture that.

Reply:
Agreed - thanks for capturing that!